src/pages/Scholarships.jsx

```diff
@@ -45,8 +49,9 @@ function ScholarshipsPage() {
         (loading && <CircularProgress className={classes.progress} />) || (
           <ScholarshipList scholarships={scholarships} />
         )}
-      <Box mt={24} mb={24} />
       <Button
+        mt={theme.spacing(3)}
+        mb={theme.spacing(3)}
```

Comment:
This doesn't actually set the margin. You can see what I mean by looking at this element via the inspector:
![image](https://user-images.githubusercontent.com/8023233/106715395-e1645d80-65b1-11eb-891a-eecfb038d541.png)

There's no margin. 

If you read [spacing docs](https://material-ui.com/system/spacing/) more carefully, you'll notice how the [input](https://material-ui.com/system/spacing/#transformation) is just a number. E.g. `3`, not `theme.spacing(3)`. `theme.spacing(3)` is used when applying styles using `makeStyles((theme) => {...})` like you technically could on line 14 in this file.